underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -110,6 +112,12 @@ public void close() throws IOException {
 
   @Override
   public OutputStream create(String path) throws IOException {
+    return create(path, new UnderFileSystemCreateOptions(mConfiguration));
+  }
+
+  @Override
+  public OutputStream create(String path, UnderFileSystemCreateOptions options) throws IOException {
+    // Block size and permission status in options are ignored when creating OSS object.
```

Comment:
This comment will get stale when new fields are added to `options`. I would simply remove it.